docs/genome_config.md

```diff
@@ -52,3 +52,12 @@ genomes:
 ```
 
 Any relative paths in the asset `path` attributes are considered relative to the genome config file.
+
+So, for example:
+
+```
+genomes:
+  hg38:                <--------------- that's the genome key.
+    bowtie2:           <--------------- that's the asset key.
+      path: blahblah   <--------------- relative to genome config. returned by get_asset('hg38', 'bowtie2'). 
```

Comment:
This makes it sound like we return the literal value in the config file; is that the intent? Right now I'd expect that only to be true if the value is a path that itself exists, independent of the genome folder.

Reply:
I've updated this now.